src/upload/system/library/transbank/composer.json

```diff
@@ -1,6 +1,12 @@
 {
     "require": {
-        "transbank/transbank-sdk": "~2.0"
+        "transbank/transbank-sdk": "~2.0",
+        "monolog/monolog": "^1.27"
+    },
+    "autoload": {
+        "psr-4": {
+            "Transbank\\Utils\\": "system/library/Transbank/utils/"
```

Comment:
What happens if in the future you want to have more namespaces? reconsider my namespace proposal